SPECS/grub2/grub2.spec

```diff
@@ -186,6 +189,13 @@ Group:          System Environment/Base
 Directory for package-specific boot configurations
 to be persistently stored on AzureLinux
 
+%package mkconfig
+Summary:        Minimal set of utilities to configure a grub-based system
+Group:          System Environment/Base
```

Comment:
shouldn't this package `Requires: grub2-configuration`? mkconfig will need some configuration to actually do anything, right?